crates/uv/src/main.rs

```diff
@@ -1012,10 +1016,17 @@ async fn run() -> Result<ExitStatus> {
                 Vec::new(),
                 args.no_index,
             );
+            let cwd = env::current_dir()?;
+            let prompt = if args.prompt == "." {
+                cwd.file_name().map(|p| p.to_str().unwrap())
```

Comment:
We try not to `unwrap()` in user-facing code, is there a safe way to get what you're looking for here?

Will this be a confusing user experience if this ends up being `None`?